### ISSUE TITLE
[Quest][Blade's Edge Mountains] To Rule The Skies

### ISSUE THREAD

**Issue by Anelith** [FeenixServerProject/Archangel_2.4.3_Bugtracker#2852]
**Current situation:** The quest **To Rule the Skies** http://www.wowhead.com/quest=11078 is pretty much broken.

**Desirued situation:**
_From wowhead:_
There are a total of four dragons that you can summon for this quest - Rivendark, Obsidia, Furywing, and Insidion.

Rivendark can be summoned at (27, 64).
Obsidia can be summoned at (34, 54).
Furywing can be summoned at (66, 14).
Insidion can be summoned at (62, 7).

All four dragons are nearly identical. They are basically Onyxia clones tailored for groups of size 3-5 people. They all possess the following abilities:

Melee Attack - hit the tank for about 450-600 damage
Cleave - standard cleave attack
Flame Breath - frontal cone attack for about 600 fire damage
Tail Swipe - knockback effect from the tail of the dragon (similar to Onyxia's). Our fury warrior kept getting knocked backwards right off the ledge.
Bellowing Roar - AoE fear, similar to Onyxia's

Along with the afore mentioned abilities, each dragon possess exactly one unique ability:

Rivendark - casts Corruption on random party members; identical to a warlock's Corruption
Obsidia - casts Hellfire; identical to a warlock's Hellfire. The Hellfire was hitting for about 500 fire damage a tick.
Furywing - casts Wing Buffet on the tank; similar to Firemaw's Wing Buffet. This ability reduces the threat on primary target by a significant amount and knocks that target back. It's not a big deal if the tank loses aggro, since Furywing is not immune to taunt.
Insidion - casts a stacking Flame Buffet on the entire party; similar to Firemaw's Flame Buffet. This ability increases fire damage taken by the party and stacks a large number of times. Since Insidion's only fire attack is Flame Breath, the only person really getting hit with fire damage should be the tank. Thus, Flame Buffet stacking on the rest of the group shouldn't really matter.

For loot, each dragon drops a green or blue item, as well as a scale unique to that dragon (lootable by everybody in the party). If you obtain a scale from each of the four dragons, you can create this item http://www.wowhead.com/?item=31942#M0z.

**In short:**
object=185937 should summon npc=23261 Furywing
object=185938 should summon npc=23281 Insidion
object=185932 should summon npc=23282 Obsidia
object=185936 should summon npc=23061 Rivendark

In additional:
Object 185937 (Ouef d'Aile-furie) is using French version in the database, it should use the English one and it should be called (Furywing's Egg)
Object 185938 (Oeuf d'Insidion) is using French version in the database, it should use the English one and it should be called (Insidion's Egg)
